src/notation/view/widgets/editstaff.cpp

```diff
@@ -156,6 +156,7 @@ void EditStaff::setStaff(Staff* s, const Fraction& tick)
     invisible->setChecked(m_staff->invisible(Fraction(0, 1)));
     isSmallCheckbox->setChecked(stt->isSmall());
     color->setColor(stt->color().toQColor());
+
```

Comment:
this change seems useless and may just lead to merge conflicts for no good reason

Reply:
Ahh yes. I missed that during rebase. I will fix it. Thnkx.

Reply:
Done